src/EnergyPlus/NodeInputManager.cc

```diff
@@ -109,8 +110,8 @@ namespace NodeInputManager {
 		NumCheckNodes = 0;
 		MaxCheckNodes = 0;
 		NodeVarsSetup = false;
-		NodeWetBulbRepReq.deallocate();
```

Comment:
I probably had a reason when I did that, but can't recall now. I guess it should go back in.